site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -2,6 +2,9 @@
 
 [.api]
 ifdef::login_response,passwordless_login_response[]
+[field]#pendingIdPLinkId# [type]#[String]# [since]#Available since 1.28.0#::
+The pending identity provider link Id. This value is created when logging in with an identity provider configured with a linking strategy of `Create a pending link`. It is used in conjunction with the link:/docs/v1/tech/apis/identity-providers/links.adoc[Link APIs] to complete a pending link.
```

Comment:
We should also describe when this will be in the response.

Reply:
Done